tables/iRO/Renewal/shuffle.txt

```diff
@@ -1,217 +1,219 @@
-# Generated: 2019-08-30T03:01:43Z
-007D 0127
-0090 0898
-00B8 0939
-0126 0437
-0127 08A1
-0140 0955
-0178 0926
-014D 0919
-014F 0952
+# Generated: 2019-09-27T14:37:50Z
+007D 08A2
+0090 086B
+00B8 0941
+0126 0932
+0127 088E
+0140 0956
+0178 0360
+014D 0886
+014F 0964
 0AF1 0821
-08C9 035F
-09CE 0945
-0A68 085F
-0A5A 0950
-0A5C 088B
-0A69 0878
-0A6C 0361
-0A6E 02C4
+08C9 0969
+09CE 085B
+0A68 0867
+0A5A 0A5C
+0A5C 0888
+0A69 0866
+0A6C 0870
+0A6E 094C
 0A0D 0A8B
+0AF1 0821
```

Comment:
Double entry.

Reply:
it need del? @Fadreus

Reply:
Yup..

Reply:
Yup..

https://github.com/OpenKore/openkore/blob/704af5edebbf2f2de43e0426b8f340313873f828/tables/iRO/Renewal/shuffle.txt#L11

https://github.com/OpenKore/openkore/blob/704af5edebbf2f2de43e0426b8f340313873f828/tables/iRO/Renewal/shuffle.txt#L21

Reply:
It seems inventory list will be empty after map switching.

Reply:
Fix 52dcbc0